priv/graphql/schemas/admin/mnesia.gql

```diff
@@ -0,0 +1,45 @@
+"""
+Allow admin to manage mnesia database
+"""
+
+type MnesiaAdminQuery @protected{
+  info(keys: [String!]!): [MnesiaInfo]
+    @protected(type: GLOBAL)
+}
+
+type MnesiaAdminMutation @protected{
```

Comment:
We should have one-sentence descriptions for each operation, right? Otherwise the CLI (and online) help would contain no description.